starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,27 @@
 // Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2){
+    return num1;
+  } else if (num1 < num2){
+      return num2;
+  } else if (num1 === num2) {
+      return num1;
+  }
+}
 
 // Finding Longest Word
+function findLongestWord(words){
+ var maxLength = 0;
+ var maxWordIndex = 0;
+  for (var i = 0; i < words.length; i++){
+    if (words[i].length > maxLength) {
+      maxLength = words[i].length;
+      maxWordIndex = i;
+    }
+ }
+ return words[maxWordIndex];
```

Comment:
Two things to keep in mind:
 - You don't need two variables. You can store the longest word in a variable called 'longestWord' and access its length with 'longestWord.length'.
 - This function will fail returning undefined when the array is empty. You can solve this checking if the array is empty and returning undefined then. A better way is asign to your comparator (longestWord) the first array's element. If the array is empty, longestWord will be 'undefined' and the function will return 'undefined'. Using an internal element for comparision solve some tricky problems (e.g. finding a minimum).
```
function findLongestWord(words) {
  var longestWord = words[0];

  for (var i = 1; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }
  return longestWord;
}
```